<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PYQTDIALOG</class>
 <widget class="QDialog" name="PYQTDIALOG">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>982</width>
    <height>611</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>查找断码</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <widget class="QPushButton" name="btn2">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>查找</string>
   </property>
  </widget>
  <widget class="QLabel" name="image">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>500</y>
     <width>101</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="btn1">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>29</x>
     <y>9</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>打开</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>101</width>
     <height>411</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWebView" name="webView">
   <property name="geometry">
    <rect>
     <x>139</x>
     <y>30</y>
     <width>841</width>
     <height>571</height>
    </rect>
   </property>
   <property name="url">
    <url>
     <string>about:blank</string>
    </url>
   </property>
  </widget>
  <widget class="QLineEdit" name="netaddr">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>0</y>
     <width>351</width>
     <height>21</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn3">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>0</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>外部浏览器</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btncp">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>0</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>复制</string>
   </property>
  </widget>
  <widget class="QLabel" name="stLabel">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>10</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKit/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
